grapher/Models/Theming/IO/ThemeFileOperations.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Xml;
+using System.Xml.Linq;
+
+namespace grapher.Models.Theming.IO
+{
+    public class ThemeFileOperations
+    {
+        public string ThemePath { get; set; }
+
+        public IEnumerable<ColorScheme> LoadThemes()
+        {
+            ThemePath = Path.Combine(Environment.CurrentDirectory, "/themes");
+
+            var pathFound = Directory.Exists(ThemePath);
+
+            if (!pathFound)
+            {
+                Directory.CreateDirectory(ThemePath);
+                CreateDefaultThemes();
+            }
+
+            var schemes = new List<ColorScheme>();
+
+            var files = Directory.GetFiles(ThemePath);
+            foreach (var filePath in files)
+            {
+                var xmlDoc = new XmlDocument();
+
+                xmlDoc.Load(filePath);
```

Comment:
This could be:
var schemes = files.Select(f => XDocument.Load(f)).Select(x => ColorSchemeManager.FromXml(x));

Reply:
Ah, thanks that makes sense, I have implemented this, but used a method group instead of using the f => func(f)